openj9.test.daa/src/test.daa/net/openj9/test/PDMoveShifts/TestShiftsAndConvert.java

```diff
@@ -2519,6 +2519,19 @@ public void testShiftLeftInvalidPrecision()
     @Test
     public void testLeftOffsetOne()
     {
+        /* Both expectedArray and expectedArray2 are needed because
```

Comment:
@VermaSh  We do not need to quote PoPs here. I think if you elaborate on behavior difference from z14 and z15 here, it would be sufficient. (I think one of the updates you posted earlier on internal issue would be more suitable).

Reply:
I have replaced the quote with a summary explaining the behavior change.